src/js/tabs/configuration.js

```diff
@@ -518,6 +518,9 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
 
         if (semver.gte(CONFIG.apiVersion, "1.31.0")) {
             $('input[name="fpvCamAngleDegrees"]').val(RX_CONFIG.fpvCamAngleDegrees);
+            if (semver.gte(CONFIG.apiVersion, "1.41.0")) {
+                $('input[name="fpvCamAngleDegrees"]').attr("max": 90);
```

Comment:
Have you tested the change? If I'm not wrong, this will not work. When using only one parameter you need to do as `.attr("max", "90")` and not `.attr("max": 90)`. Maybe jquery is intelligent enough and accept both ways?

Reply:
No, damn, I haven't.